cw/capture.py

```diff
@@ -1194,6 +1194,17 @@ def capture_otbn_vertical(ot, ktp, fw_bin, pll_frequency, capture_cfg):
         check_range(waves, ot.scope.adc.bits_per_sample)
         project.traces.append(trace, dtype=np.uint16)
 
+    # Save metadata to project file
+    project.settingsDict['pll_frequency'] = pll_frequency
+    project.settingsDict['sample_rate'] = int(round(ot.scope.clock.clkgen_freq, -6))
+    project.settingsDict['masks_off'] = capture_cfg["masks_off"]
+    project.settingsDict['num_samples'] = capture_cfg["num_samples"]
+    project.settingsDict['offset'] = capture_cfg["offset"]
+    project.settingsDict['scope_gain'] = capture_cfg["scope_gain"]
+    project.settingsDict['num_traces'] = capture_cfg["num_traces"]
+    # store last number of cycles where the trigger signal was high to metadata
+    project.settingsDict['samples_trigger_high'] = cycles
+
```

Comment:
Maybe we can also add date/time. This may help us to differentiate the results from each other.

Reply:
Good idea!